data.sql

```diff
@@ -46,4 +46,41 @@ WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 
 UPDATE animals
 SET owner_id = (SELECT id FROM owners WHERE fullname = 'Dean Winchester')
-WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+WHERE name IN ('Angemon', 'Boarmon');
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+('Williams Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vets_id, species_id)
+VALUES
+(1, 1),
+(2, 2),
+(3, 1),
+(4, 2);
+
+INSERT INTO visits (animal_id, vets_id, date_of_visits)
+VALUES
+(1, 1, '2020-05-24'),
+(1, 3, '2020-07-22'),
+(2, 4, '2021-02-02'),
+(5, 2, '2020-01-05'),
+(5, 2, '2020-03-08'),
+(5, 2, '2020-05-14'),
```

Comment:
- [ ] Let us kindly take a look at `line 70-72`. Hardcoded (5,2), animal_id = 5 in your data is `'Charmander'` instead of `'Pikachu'` which is animal_id = 3. Let us take time and go through and make the right data insertion as seen in the screenshot below
![Screenshot from 2023-05-08 08-49-37](https://user-images.githubusercontent.com/46388113/236778604-2b284670-034a-4ee0-bd27-116886b5befc.png)

Let us check all the other data as well before submitting another code review request